com.trekglobal.idempiere.rest.api/src/com/trekglobal/idempiere/rest/api/json/RestUtils.java

```diff
@@ -170,12 +170,21 @@ public static Query getQuery(String tableName, String whereClause, List<Object>
 			whereClause = whereClause + "AD_Language=?";
 			params.add(Env.getAD_Language(Env.getCtx()));
 		}
+		
+		if(includeInactive!=null && includeInactive.equals("only")) {
+			whereClause += "isActive=?";
```

Comment:
There is no check here in case the whereClause is empty or null (like in line 167)